src/palindrom-dom.js

```diff
@@ -10,29 +10,43 @@
    * @param {Object} [options] map of arguments. See README.md for description
    */
   var PalindromDOM = function (options){
-    options || (options={});
+    if (typeof options !== "object") {
+            throw new Error("'options' is not an object");
```

Comment:
I'd either do more descriptive message like "Palindorm() requires and object argument...." as by running `new Palindrom()` you have no idea what are the `options`. Or  skip it totally, and reduce it to 
`if(!(options && options.remoteUrl)) { throw new Error('remoteUrl option is required');`